release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -85,6 +85,28 @@ CRI-O brings:
 
 image::crio-3-7.png[CRI-O]
 
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=True
+----
+
+This setting pulls the *openshift3/cri-o:latest* image from the link:https://access.redhat.com/containers[Red Hat Registry]. It also sets
+`openshift_docker_systemcontainer_image_registry_override` as its override just
+as the *container-engine* image does (as seen in
+xref:../install_config/install/advanced_install.adoc#advanced-install-docker-system-container[Running Docker as a System Container].)
+
+[NOTE]
+====
+{product-title} node hosts must be RPM-based or system containers when using
```

Comment:
This is probably fine, but it's really the ``atomic-enterprise-node``/``origin-node`` service that can not be a docker container (but can be rpm or system container installed).